.gitignore

```diff
@@ -167,3 +164,8 @@ cython_debug/
 
 # VSCode
 .vscode/
+
+# Ragna
+ragna.toml
+ragna/_version.py
```

Comment:
Can we keep this at the top please? These are Ragna specific things and should be easily discoverable.